templates/forms/FormField_holder.ss

```diff
@@ -1,4 +1,4 @@
-<div id="$HolderID" class="field<% if $extraClass %> $extraClass<% end_if %>">
+<div id="$HolderID" class="field<% if $extraClass %> $extraClass<% end_if %><% if $Required %> requiredField<% end_if %>">
```

Comment:
This could also be achieved by adding an extension on FormField that adds `requiredField` as an extra CSS class via `if ($formField->Required()) { $formField->addExtraClass('requiredField'); }`.

If this was going to be made a default CSS class for form fields I think it should probably be done in `FormField::extraClass` - there's an example there already of it adding `nolabel` when there's no title on the field.